docs/segment.md

```diff
@@ -2,98 +2,188 @@
 title: Segment with eScriptorium
 summary: About segmentation features in eScriptorium.
 authors:
-    - Alix Chagué, Floriane Chiffoleau
+    - Alix Chagué, Floriane Chiffoleau, Hugo Scheithauer
 date: 2023-04-07
 ---
 
 # Segment with eScriptorium
 
-<!-- todo:
+This section introduces the **segmentation panels** and some related features. The segmentation panel is one of the panels available in the "Edit" tab, once you are in a document (`{base_url}/document/{document-id}/images/`).
 
-This section should provide solid definition of the elements involved in segmentation (masks/polygons, baselines/toplines, zones, reading order, reading direction, type for region and lines, line ordering (partially covered in transcribe).)
+The segmentation panel is toggled by clicking on the "Segmentation" button at the top of the page (or by pressing ++control+3++).
+
+![image: Navigating to the segmentation panel in the edit tab](img/segment/segmentation_panel.gif "The 3rd panel is used to manually edit the segmentation")
 
-It should also make sure that the importance of when segmentation is done is understood (risk of erasing data if done after a round of transcription), and mention the fact that segmentation can result from prediction, manual segmentation or import.
+Segmentation, also called zoning, layout analysis, document analysis, or even optical layout analysis, refers to two processes:
 
--->
+- Extracting and labeling the structure of a document, such as text, images, tables, columns, headings, footers, etc. The goal is to segment the document into meaningful regions, or bounding boxes, that can be further processes or analyzed. This step is optional for transcription, but can be important for later post-processing in an editorial or archiving context.
 
-Segmentation consists in locating lines of text on the image and identifying the layout. Segmentation includes drawing **baselines** <!-- todo: add a definition -->, **polygones** (or masks) <!-- todo: add a definition --> and **regions** (or zones)<!--todo: add a definition -->. Such 
+- Precisely detecting lines of text in an image, which is mandatory in eScriptorium for transcribing said lines later.
 
-It is possible to perform [segmentation automatically](predict.md#predict-the-segmentation) or manually. This section covers how to create segmentation information from scratch as well as how to modify an existing segmentation.
+Segmentation can result from a prediction, can be performed manually, or can be imported with the **import** button when you import an XML file directly from the **Images** tab inside a document.
```

Comment:
I think you should add again the link to predict-the-segmentation and also add a link to the import-the-segmentation